db/fixtures/miq_event_definitions.csv

```diff
@@ -159,3 +159,9 @@ request_service_start,Service Start Request,Default,service_process
 service_started,Service Started,Default,service_process
 request_service_stop,Service Stop Request,Default,service_process
 service_stopped,Service Stopped,Default,service_process
+
+#
+# Container Operations
+#
+containerimage_scan_complete,Container Image Analysis Complete,Default,container_operations
+containerimage_created,New Container Image created,Default,container_operations
```

Comment:
@gmcculloug Could you explain what the request_X events are for? 
I believe we currently do not use such events for container ssa but maybe we should.